stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -891,12 +974,32 @@ ::util::StatusOr<int> BFChassisManager::GetUnitFromNodeId(
   return *unit;
 }
 
+::util::StatusOr<uint32> BFChassisManager::GetSdkPortId(
+    uint64 node_id, uint32 port_id) const {
+  if (!initialized_) {
+    return MAKE_ERROR(ERR_NOT_INITIALIZED) << "Not initialized!";
+  }
+  const int* unit = gtl::FindOrNull(node_id_to_unit_, node_id);
+  CHECK_RETURN_IF_FALSE(unit != nullptr)
+      << "Node " << node_id << " is not configured or not known.";
```

Comment:
`unit` is not used later?

Reply:
Was used originally, but replaced call with map lookup and never cleaned this up. fixed